src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using NosCore.Algorithm.HpService;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NosCore.Algorithm.DamageService
+{
+    // Work in progress
+    public class DamageMinMaxService : IDamageMinMaxService
```

Comment:
I think DamageService and IDamageService are better name

Reply:
I use DamageMinMax cause i want to add on the same folder the HitRate, and the secondary stat

Reply:
Don’t use the same folder use a different service